protocol/v2/qbft/instance/instance.go

```diff
@@ -4,18 +4,14 @@ import (
 	"encoding/json"
 	"sync"
 
-	ipfslog "github.com/ipfs/go-log"
-	"github.com/pkg/errors"
-	"go.uber.org/zap"
-
 	specqbft "github.com/bloxapp/ssv-spec/qbft"
 	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
 
 	"github.com/bloxapp/ssv/protocol/v2/qbft"
 )
 
-var logger = ipfslog.Logger("ssv/protocol/qbft/instance").Desugar()
```

Comment:
same comment as above

Reply:
@aaron-blox There's an issue that fields which are necessary for logging are encoded in `StartValue`, so we need to decode it to get them, which breaks the abstraction since `StartValue`'s type is `[]byte` and technically can be anything using any encoding